content/reliability/docs/architecting-for-static-stability.md

```diff
@@ -0,0 +1,19 @@
+# Architecting for static stability
+
+When an availability zone outage occurs, the best way to maintain application availability is to be prepared. Rely on Kubernetes’ static stability, rather than automated evacuation strategies. Configure applications with [pod anti affinity](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#affinity-and-anti-affinity) and [topology spread constraints](https://kubernetes.io/docs/concepts/scheduling-eviction/topology-spread-constraints) to spread replicas across failure domains, and test to verify that replicas can handle the traffic surge during an outage.
```

Comment:
Let's add line or two defining what static stability means. Something like "In a statically stable design, the overall system keeps working even when a dependency becomes impaired."

Reply:
```suggestion
When an Availability Zone outage occurs, the best way to maintain application availability is to be prepared. Rely on Kubernetes’ static stability, rather than automated evacuation strategies. Configure applications with [pod anti affinity](https://kubernetes.io/docs/concepts/scheduling-eviction/assign-pod-node/#affinity-and-anti-affinity) and [topology spread constraints](https://kubernetes.io/docs/concepts/scheduling-eviction/topology-spread-constraints) to spread replicas across failure domains, and test to verify that replicas can handle the traffic surge during an outage.
```